src/prog_algs/metrics/samples.py

```diff
@@ -88,7 +88,7 @@ def mean_square_error(values, ground_truth):
     Returns:
         float: mean square error of eol predictions
     """
-    return sum([(x.mean() - ground_truth)**2 for x in values])/len(values)
+    return sum([(sum(x)/len(x) - ground_truth)**2 for x in values])/len(values)
```

Comment:
This change supports the case where x is not unweighted_samples, but instead is a list (some users keep doing that).

Reply:
I updated to use numpy::mean so it'll be compatible with the upcoming vectorization changes.